tests/models/gptsan_japanese/test_modeling_gptsan_japanese.py

```diff
@@ -0,0 +1,327 @@
+                -7.55021333694458,
+                -6.49869966506958,
+                -7.586796283721924,
+                -6.978085994720459,
+                -7.839145183563232,
+                -8.21964168548584,
+                -8.695091247558594,
+                -6.706910610198975,
+                -6.6585798263549805,
+                -19.565698623657227,
+                -5.353842735290527,
+                -8.350686073303223,
+                -8.039388656616211,
+                -10.856569290161133,
+                -7.75154447555542,
+                -8.819022178649902,
+                -19.51532745361328,
+            ],
+            [
+                -9.73066234588623,
+                -10.223922729492188,
+                -9.932981491088867,
+                -11.857836723327637,
+                -7.662626266479492,
+                -11.13529109954834,
+                -7.765097618103027,
+                -11.472923278808594,
+                -9.543149948120117,
+                -11.905633926391602,
+                -9.366164207458496,
+                -11.5734281539917,
+                -23.699003219604492,
+                -9.429590225219727,
+                -10.42839241027832,
+                -10.585240364074707,
+                -10.94771957397461,
+                -11.095416069030762,
+                -10.390240669250488,
+                -23.769372940063477,
+            ],
+            [
+                -9.728265762329102,
+                -9.859712600708008,
+                -10.09729290008545,
+                -9.678522109985352,
+                -6.879519939422607,
+                -9.68487548828125,
+                -4.2803425788879395,
+                -10.018914222717285,
+                -9.308445930480957,
+                -10.63394546508789,
+                -8.083646774291992,
+                -9.06301498413086,
+                -21.904266357421875,
+                -8.90160846710205,
+                -8.841876029968262,
+                -11.856719970703125,
+                -12.079398155212402,
+                -11.233753204345703,
+                -10.177338600158691,
+                -21.87256622314453,
+            ],
+            [
+                -9.669764518737793,
+                -9.614198684692383,
+                -9.814510345458984,
+                -9.996501922607422,
+                -11.375690460205078,
+                -10.113405227661133,
+                -10.546867370605469,
+                -10.04369068145752,
+                -10.907809257507324,
+                -10.504216194152832,
+                -11.129199028015137,
+                -10.151124000549316,
+                -21.96586799621582,
+                -9.086349487304688,
+                -11.730339050292969,
+                -10.460667610168457,
+                -10.298049926757812,
+                -10.784148216247559,
+                -10.840693473815918,
+                -22.03152847290039,
+            ],
+        ]
```

Comment:
You can use `#fmt off/on` for this. This will reduce the number of lines taken by that